src/build/vita-public.h

```diff
@@ -0,0 +1,9 @@
+#ifndef VITA_PUBLIC
```

Comment:
I'm not fond of this name because it somehow implies that this is vita-related, which it isn't. Maybe `FUNC_PUBLIC` instead?